### DISCUSSION TITLE
Deployment problems

### DISCUSSION THREAD

**Discussion by a2jinhee** [community/community#44491] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi! I'm currently using Jekyll Jasper(https://github.com/jekyllt/jasper) theme for my gh pages, but running into some errors. 

The main index page is deployed just fine, but for some reason when I access the pages in a subdirectory (ex. tab/speech) a 404 error pops up. Also, when the sites are approached locally it is fine. The subpages just won't be processed remotely. 

I've been at this for hours, so please help me !

**Comment by LAPCoder** [comment]
Hey @a2jinhee

I'm not very familiar with GitHub pages themes, but you should change in https://github.com/a2jinhee/a2jinhee.github.io/blob/main/_config.yml parameters like `short_url`, `repo` or `username`.
I hope this is helpful to you.

**Reply by a2jinhee** [reply]
Thanks for the answer! I tried this, (and I'm sure it changed something somewhere) but it still brings a 404 error :(

**Comment by LAPCoder** [comment]
I see the problem. You copied only the https://github.com/jekyllt/jasper/tree/master branch. The whole site can be found in the branch https://github.com/jekyllt/jasper/tree/gh-pages. 

BUT your site is a copy of the master branch (= The doc, README…) but not the gh-pages branch (= the demo website).

The [about page](https://a2jinhee.github.io/about/) is on your main branch. That file is deployed on your site. But the other pages ***doesn't exist** on your repo*!

You should create an `my_projects` folder, for example, and create a `index.html`/`index.md` file inside.
(You can also create `a/b/test/index.html`/`index.md` file).

Create the other pages from [here](https://github.com/a2jinhee/a2jinhee.github.io/blob/main/about/index.md) or similar.
